src/test/java/com/easypost/TestUtils.java

```diff
@@ -19,7 +19,7 @@ public enum ApiKey {
     }
 
     private static final String API_KEY_FAILED_TO_PULL = "couldnotpullapikey";
-    private static final String CASSETTES_FOLDER = "cassettes";
+    private static final String CASSETTES_FOLDER = "src/test/java/com/easypost/cassettes";
```

Comment:
Nit: I would have put them in `src/test`, not down into the actual test module `com.easypost`. Won't block.

Reply:
I'm game, moved.